roles/openshift_node_dnsmasq/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dns_domain: 'cluster.local'
+kube_svc_ip: '172.30.0.1'
```

Comment:
Instead of using defaults here `openshift_node_dnsmasq` should just use the `openshift.node.dns_domain` and `openshift.node.kube_svc_ip` values, since it has a dependency on `openshift_common`.